ocs_ci/helpers/helpers.py

```diff
@@ -4480,3 +4486,97 @@ def verify_log_exist_in_pod_logs(
     )
     logger.info(f"logs osd:{pod_logs}")
     return expected_log in pod_logs
+
+
+def retrieve_cli_binary(cli_type="mcg"):
+    """
+    Download the MCG-CLI/ODF-CLI binary and store it locally.
+
+    Args:
+        cli_type (str): choose which bin file you want to download ["odf" -> odf-cli , "mcg" -> mcg-cli]
+
+    Raises:
+        AssertionError: In the case the CLI binary is not executable.
+
+    """
+    semantic_version = version.get_semantic_ocs_version_from_config()
```

Comment:
In case of ODF version is < 4.15 and cli_type == "odf" please raise an exception that it's not supported

Reply:
done